src/psyclone/psyir/backend/fortran.py

```diff
@@ -202,6 +205,44 @@ def gen_vardecl(self, symbol):
         result += "\n"
         return result
 
+    def gen_decls(self, symbol_table, args_allowed=True):
+        '''Create and return the Fortran declarations for this SymbolTable.
+
+        :param symbol_table: the SymbolTable instance.
+        :type symbol: :py:class:`psyclone.psyGen.SymbolTable`
+        :param bool args_allowed: if False then one or more argument
+        declarations in symbol_table will cause this method to raise
+        an exception. Defaults to True.
+
+        :returns: the Fortran declarations as a string.
+        :rtype: str
+
```

Comment:
Please document the exception that this routine can raise.

Reply:
Sorry, I forgot to add this for some reason.